examples/apple/coreml/README.md

```diff
@@ -27,40 +27,50 @@ cd executorch
 
 ```
 
-3. Run the export script to generate a **Core ML** delegated binary file.
+3. Run the export script to generate a Core ML delegated binary file.
 
 ```bash
 cd executorch
 
 # To get a list of example models
 python3 -m examples.portable.scripts.export -h
 
-# Generates ./add_coreml_all.pte file if successful.
+# Generates add_coreml_all.pte file if successful.
 python3 -m examples.apple.coreml.scripts.export --model_name add
 ```
 
-4. Once we have the **Core ML** delegated model binary (pte) file, then let's run it with the **ExecuTorch** runtime using the `coreml_executor_runner`.
+4. Run the binary file using the `coreml_executor_runner`.
 
 ```bash
 cd executorch
 
 # Builds the Core ML executor runner. Generates ./coreml_executor_runner if successful.
 ./examples/apple/coreml/scripts/build_executor_runner.sh
 
-# Run the Core ML delegate model.
+# Run the delegated model.
 ./coreml_executor_runner --model_path add_coreml_all.pte
 ```
 
 ## Frequently encountered errors and resolution.
-- The `examples.apple.coreml.scripts.export` could fail if the model is not supported by the **Core ML** backend. The following models from the examples models list (` python3 -m examples.portable.scripts.export -h`)are currently supported by the **Core ML** backend.
+- The `examples.apple.coreml.scripts.export` could fail if the model is not supported by the Core ML backend. The following models from the examples models list (` python3 -m examples.portable.scripts.export -h`) are currently supported by the Core ML backend.
 
-```
+```text
 add
 add_mul
+dl3
+edsr
+emformer_join
+emformer_predict
+emformer_transcribe
+ic3
 ic4
 linear
+llama2
```

Comment:
have you tried llama3? If llama2 works, llama3 should be just out of box.

Reply:
Not yet, going to try today.